src/Domain/Stager.php

```diff
@@ -61,11 +85,11 @@ public function stage(array $command, string $stagingDir): void
      */
     private function validateCommand(): void
     {
-        if ($this->command === []) {
+        if ($this->composerCommand === []) {
```

Comment:
Nit: if the command cannot be empty, maybe `empty($this->composerCommand)` would be a more robust check here.

Reply:
I'm not sure what you mean by "robust", but PHPStan complains about `empty()` here for type safety. (I used `empty()` first, too.)

```
 ------ ---------------------------------------------------------------
  Line   src/Domain/Stager.php
 ------ ---------------------------------------------------------------
  88     Construct empty() is not allowed. Use more strict comparison.
 ------ ---------------------------------------------------------------
```

Reply:
Hah! That is, to me, a nutty opinion for PHPStan to have. `empty()` is the best and most concise way to capture as many forms of "emptiness" as possible. Sure, you COULD do this:

```
if ($foo === [] || $foo === false || $foo === '' || $foo === 0 || $foo === 0.0 || $foo === '0' || $foo === '0.0' || $foo === null)
```

Or you could just freakin' use `empty($foo)` to check for ALL of those cases! If I ever meet PHPStan in the street, I'm going to slap it.